plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -116,7 +124,7 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
 
                 // Phase 3
                 val properties = symbol.propertySymbols
-                newEmbedding.initFields(properties.mapNotNull { processBackingFields(it, newEmbedding) }.toMap())
+                newEmbedding.initFields(properties.mapNotNull { processBackingFields(it, symbol) }.toMap())
```

Comment:
Was this a bug, or is this a behaviour change?

Reply:
I just needed a class symbol to check its finality in the function, but I don't need the `TypeEmbedding` now.
I wasn't sure if `FirPropertySymbol.isFinal` always returns true if class is final.